earth_enterprise/src/fusion/autoingest/sysman/AssetVersionD.cpp

```diff
@@ -187,7 +200,14 @@ AssetVersionImplD::SetState(AssetDefs::State newstate, bool propagate)
     // set it to above. OnStateChange can call SetState recursively. We
     // don't want to notify/propagate an old state.
     if (propagate && (state == newstate)) {
+      notify(NFY_VERBOSE, "We wish to propagate and state: %s is equal to newstate: %s", 
+             ToString(state).c_str(), 
+             ToString(newstate).c_str());
+      notify(NFY_VERBOSE, "Calling theAssetManager.NotifyVersionStateChange(%s, %s)", 
+             GetRef().c_str(), 
+             ToString(newstate).c_str());
       theAssetManager.NotifyVersionStateChange(GetRef(), newstate);
+      notify(NFY_VERBOSE, "Calling PropagateStateChange()");
```

Comment:
PropagateStateChange() has an entrance notification.  This notification is probably unnecessary